test/unit/spec/EnrollWindowsHello_spec.js

```diff
@@ -187,6 +189,42 @@ function (Okta,
           });
         });
       });
+
+      itp('does not show error if webauthn.makeCredential fails with AbortError', function () {
+        return emulateWindows('AbortError')
+        .then(setup)
+        .then(function (test) {
+          test.setNextResponse([responseMfaEnrollActivateWebauthn, responseSuccess]);
+          test.form.submit();
+          return tick(test);
+        })
+        .then(function (test) {
+          expect(webauthn.makeCredential).toHaveBeenCalled();
+          return tick(test);
+        })
+        .then(function (test) {
+          expect($.ajax.calls.count()).toBe(2);
+          expect(test.form.hasErrorWindowsHello()).toBe(false);
+        });
+      });
+
+      itp('shows error if webauthn.makeCredential fails with NotSupportedError', function () {
+        return emulateWindows('NotSupportedError')
+        .then(setup)
+        .then(function (test) {
+          test.setNextResponse([responseMfaEnrollActivateWebauthn, responseSuccess]);
+          test.form.submit();
+          return tick(test);
```

Comment:
Same here (and for all the following tests) - there are no longer very many cases where we should actually use a tick() - check out Expect.js for the different wait conditions.